search-server/spacewalk-search/spacewalk-search.spec

```diff
@@ -88,7 +88,7 @@ Obsoletes:      rhn-search < 5.3.0
 %if 0%{?fedora} || 0%{?rhel} >=7
 Requires:       mchange-commons
 %endif
-%if 0%{?fedora} >= 21
+%if 0%{?fedora} >= 21 || 0%{?sle_version} >= 150200
```

Comment:
It's funny that we had this for `fedora >= 21` already.

It seems that the JAR for log4j has the same name there, no matter the version. Otherwise I don't understand how the original `build-props.xml` worked :-|